packages/core/base/src/services/crossmintPaymentService.ts

```diff
@@ -1,9 +1,10 @@
+import { FiatPaymentElementProps, PaymentElementProps } from "../types";
 import { CrossmintCheckoutEventUnion, ParamsUpdatePayload, PaymentElementSDKEvents } from "../types/events";
 import { CheckoutEvents } from "../types/events";
-import { PaymentElement } from "../types/paymentElement";
 import { getEnvironmentBaseUrl } from "../utils";
 
-export function crossmintPaymentService(props: PaymentElement) {
+// TODO: Accept both fiat and crypto
+export function crossmintPaymentService(props: FiatPaymentElementProps) {
```

Comment:
The reference for the new component is going to be  determined by this function, or an internal redirect on 

`/sdk/paymentElement`?

Reply:
sort of both

this function will return the correct iframe url (so, the new route only for crypto pay currently), but then we will also have different components